### ISSUE TITLE
I get ImportError: DLL load failed: The specified procedure could not be found when i import pandas_profiling

### ISSUE THREAD

**Issue by Amrouane** [pandas-profiling/pandas-profiling#266]
I Get the error:  **ImportError: DLL load failed: The specified procedure could not be found.** whenever i try to import `pandas_profiling`
I am running the code on python 3.7 with package versions:
`pandas`== 0.24.2
`pandas-profiling `==2.3.0
The error log I'm getting is: 

---------------------------------------------------------------------------

```
ImportError                               Traceback (most recent call last)
<ipython-input-1-dde8e3d95713> in <module>
      1 import pandas as pd
----> 2 import pandas_profiling
      3 import numpy as np
      4 
      5 # importing the data

~\Anaconda3\lib\site-packages\pandas_profiling\__init__.py in <module>
     16 
     17 from pandas_profiling.config import config
---> 18 from pandas_profiling.controller import pandas_decorator
     19 import pandas_profiling.view.templates as templates
     20 from pandas_profiling.model.describe import describe as describe_df

~\Anaconda3\lib\site-packages\pandas_profiling\controller\pandas_decorator.py in <module>
      1 """This file add the decorator on the DataFrame object."""
      2 from pandas import DataFrame
----> 3 from pandas_profiling.__init__ import ProfileReport
      4 
      5 

~\Anaconda3\lib\site-packages\pandas_profiling\__init__.py in <module>
     18 from pandas_profiling.controller import pandas_decorator
     19 import pandas_profiling.view.templates as templates
---> 20 from pandas_profiling.model.describe import describe as describe_df
     21 from pandas_profiling.view.notebook import display_notebook_iframe
     22 from pandas_profiling.view.report import to_html

~\Anaconda3\lib\site-packages\pandas_profiling\model\describe.py in <module>
     22 from pandas_profiling.model import base
     23 from pandas_profiling.model.base import Variable
---> 24 from pandas_profiling.model.correlations import (
     25     calculate_correlations,
     26     perform_check_correlation,

~\Anaconda3\lib\site-packages\pandas_profiling\model\correlations.py in <module>
      9 from confuse import NotFoundError
     10 from pandas.core.base import DataError
---> 11 from scipy import stats
     12 
     13 from pandas_profiling.config import config

~\Anaconda3\lib\site-packages\scipy\stats\__init__.py in <module>
    365 from __future__ import division, print_function, absolute_import
    366 
--> 367 from .stats import *
    368 from .distributions import *
    369 from .morestats import *

~\Anaconda3\lib\site-packages\scipy\stats\stats.py in <module>
    171 from scipy._lib._util import _lazywhere
    172 import scipy.special as special
--> 173 from . import distributions
    174 from . import mstats_basic
    175 from ._stats_mstats_common import _find_repeats, linregress, theilslopes, siegelslopes

~\Anaconda3\lib\site-packages\scipy\stats\distributions.py in <module>
      8 from __future__ import division, print_function, absolute_import
      9 
---> 10 from ._distn_infrastructure import (entropy, rv_discrete, rv_continuous,
     11                                     rv_frozen)
     12 

~\Anaconda3\lib\site-packages\scipy\stats\_distn_infrastructure.py in <module>
     22 
     23 # for root finding for discrete distribution ppf, and max likelihood estimation
---> 24 from scipy import optimize
     25 
     26 # for functions of continuous distributions (e.g. moments, entropy, cdf)

~\Anaconda3\lib\site-packages\scipy\optimize\__init__.py in <module>
    385 
    386 from .optimize import *
--> 387 from ._minimize import *
    388 from ._root import *
    389 from ._root_scalar import *

~\Anaconda3\lib\site-packages\scipy\optimize\_minimize.py in <module>
     28 from ._trustregion_krylov import _minimize_trust_krylov
     29 from ._trustregion_exact import _minimize_trustregion_exact
---> 30 from ._trustregion_constr import _minimize_trustregion_constr
     31 
     32 # constrained minimization

~\Anaconda3\lib\site-packages\scipy\optimize\_trustregion_constr\__init__.py in <module>
      2 
      3 
----> 4 from .minimize_trustregion_constr import _minimize_trustregion_constr
      5 
      6 __all__ = ['_minimize_trustregion_constr']

~\Anaconda3\lib\site-packages\scipy\optimize\_trustregion_constr\minimize_trustregion_constr.py in <module>
      2 import time
      3 import numpy as np
----> 4 from scipy.sparse.linalg import LinearOperator
      5 from .._differentiable_functions import VectorFunction
      6 from .._constraints import (

~\Anaconda3\lib\site-packages\scipy\sparse\linalg\__init__.py in <module>
    115 from .dsolve import *
    116 from .interface import *
--> 117 from .eigen import *
    118 from .matfuncs import *
    119 from ._onenormest import *

~\Anaconda3\lib\site-packages\scipy\sparse\linalg\eigen\__init__.py in <module>
      9 from __future__ import division, print_function, absolute_import
     10 
---> 11 from .arpack import *
     12 from .lobpcg import *
     13 

~\Anaconda3\lib\site-packages\scipy\sparse\linalg\eigen\arpack\__init__.py in <module>
     20 from __future__ import division, print_function, absolute_import
     21 
---> 22 from .arpack import *

~\Anaconda3\lib\site-packages\scipy\sparse\linalg\eigen\arpack\arpack.py in <module>
     43 __all__ = ['eigs', 'eigsh', 'svds', 'ArpackError', 'ArpackNoConvergence']
     44 
---> 45 from . import _arpack
     46 import numpy as np
     47 import warnings

ImportError: DLL load failed: The specified procedure could not be found.
```

**Comment by cemsa1992** [comment]
I'm getting the same. Can use it as impport and df.profile report().

Can not use as external tool.

Error log below.

C:\Users\cemsa\AppData\Local\Continuum\anaconda3\envs\baseclone\Scripts\pandas_profiling.exe C:\Users\cemsa\Dropbox\PyCharm\Directories\AB_NYC_2019.csv C:\Users\cemsa\Dropbox\PyCharm\Directories/AB_NYC_2019_report.html
Traceback (most recent call last):
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\cemsa\AppData\Local\Continuum\anaconda3\envs\baseclone\Scripts\pandas_profiling.exe\__main__.py", line 5, in <module>
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\site-packages\pandas_profiling\__init__.py", line 18, in <module>
    from pandas_profiling.controller import pandas_decorator
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\site-packages\pandas_profiling\controller\pandas_decorator.py", line 3, in <module>
    from pandas_profiling.__init__ import ProfileReport
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\site-packages\pandas_profiling\__init__.py", line 20, in <module>
    from pandas_profiling.model.describe import describe as describe_df
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\site-packages\pandas_profiling\model\describe.py", line 24, in <module>
    from pandas_profiling.model.correlations import (
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\site-packages\pandas_profiling\model\correlations.py", line 11, in <module>
    from scipy import stats
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\site-packages\scipy\stats\__init__.py", line 367, in <module>
    from .stats import *
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\site-packages\scipy\stats\stats.py", line 172, in <module>
    import scipy.special as special
  File "c:\users\cemsa\appdata\local\continuum\anaconda3\envs\baseclone\lib\site-packages\scipy\special\__init__.py", line 641, in <module>
    from ._ufuncs import *
ImportError: DLL load failed: The specified module could not be found.

**Comment by Amrouane** [comment]
Fixed: I downgraded Scipy version to 1.3.1 and it worked fine
